pkg/api/test/client.go

```diff
@@ -0,0 +1,84 @@
+package client
+
+import (
+	"context"
+	"time"
+
+	"github.com/kyma-incubator/octopus/pkg/apis"
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8sRestClient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+type TestRESTClient interface {
```

Comment:
This client should be embedded into the Kyma Kubernetes client following the design on the Kubernetes API.

Reply:
Fixed. See `/pkg/api`